fast64_internal/oot/actor/properties.py

```diff
@@ -110,22 +163,189 @@ def draw_props(
 class OOTActorProperty(PropertyGroup):
     actorID: EnumProperty(name="Actor", items=ootData.actorData.ootEnumActorID, default="ACTOR_PLAYER")
     actorIDCustom: StringProperty(name="Actor ID", default="ACTOR_PLAYER")
+
     actorParam: StringProperty(name="Actor Parameter", default="0x0000")
     rotOverride: BoolProperty(name="Override Rotation", default=False)
-    rotOverrideX: StringProperty(name="Rot X", default="0")
-    rotOverrideY: StringProperty(name="Rot Y", default="0")
-    rotOverrideZ: StringProperty(name="Rot Z", default="0")
+    rotOverrideX: StringProperty(name="Rot X", default="0x0000")
+    rotOverrideY: StringProperty(name="Rot Y", default="0x0000")
+    rotOverrideZ: StringProperty(name="Rot Z", default="0x0000")
+
+    params: StringProperty(
+        name="Actor Parameter",
+        default="0x0000",
+        get=lambda self: self.getParamValue("Params"),
+        set=lambda self, value: self.setParamValue(value, "Params"),
+    )
+    isRotX: BoolProperty(name="", default=False, get=lambda self: self.isRotation("XRot"))
+    isRotY: BoolProperty(name="", default=False, get=lambda self: self.isRotation("YRot"))
+    isRotZ: BoolProperty(name="", default=False, get=lambda self: self.isRotation("ZRot"))
+    rotX: StringProperty(
```

Comment:
what do `isRotX` and co. mean?

Reply:
that's an internal prop used to draw the string props for X, Y or Z rots depending on what the actor is using (defined by XML data)

Reply:
so like `isRotXForActorConfig` or smth idk?

Reply:
when ``isRotX`` is true it means "this actor is using parameters inside this rotation", the name is too confusing, I'm gonna rename it ``isRotXUsedByActor``